sky/cli.py

```diff
@@ -2174,17 +2187,18 @@ def logs(
         # it will return {None: None}.
         if job_id is None:
             click.secho(f'No job found on cluster {cluster!r}.', fg='red')
-            sys.exit(1)
+            sys.exit(job_lib.JobExitCode.NOT_FOUND)
```

Comment:
minor: how about all the exceptions and exit code in the `exceptions` module? This also helps reduce the chance of circular imports.

Reply:
Good idea, refactored